websockets/websocket-next/src/test/java/io/quarkuss/ts/websocketNext/TlsWebSocketIT.java

```diff
@@ -0,0 +1,96 @@
+package io.quarkuss.ts.websocketNext;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * All TLS related stuff for websocket is moved to separate child class, since it does not work on OCP
```

Comment:
Please add todo/fixme for https://github.com/quarkus-qe/quarkus-test-framework/issues/1052

Reply:
Added